Add section about configuring user authentication in Docker Compose (#692)
memgraph/documentation: pages/getting-started/install-memgraph/docker-compose.mdx

> Want to see applications built with Memgraph and Docker Compose? Check out
> [Memgraph's Github](https://github.com/memgraph) repositories.

## Configuring user authentication in Docker Compose

> When setting up Memgraph with Docker Compose, you can use environment variables
> to manage user authentication. You can use `MEMGRAPH_USER` and
> `MEMGRAPH_PASSWORD` to either create a new user or connect with existing
> credentials.

Reviewer: `MEMGRAPH_USER` and `MEMGRAPH_PASSWORD` env vars only create user if it doesn't already exist. When it comes to connection, you still have to manually enter the credentials when trying to access Memgraph. 

```suggestion
When setting up Memgraph with Docker Compose, you can use environment variables
 `MEMGRAPH_USER` and
`MEMGRAPH_PASSWORD` to create a new user if that username doesn't already exist.
```
